Storage/Processor/Image.php

```diff
@@ -38,7 +38,7 @@ public function generate(string $filename)
             $image = $this->applyBackground($image, $width, $height);
         }
 
-        if (null !== $this->config->getRadius()) {
+        if (($this->config->getRadius() ?? 0) > 0) {
```

Comment:
This kind of nesting in the if test makes readability harder. Why not return 0 in getRadius by default in stead of null? Null would never make sense for a radius.

Reply:
Agree. Solved